packages/@aws-cdk/aws-cloudtrail/lib/index.ts

```diff
@@ -149,14 +149,13 @@ export class CloudTrail extends cdk.Construct {
       });
       this.cloudWatchLogsGroupArn = logGroup.logGroupArn;
 
-      const logsRole = new iam.Role(this, 'LogsRole', {assumedBy: new iam.ServicePrincipal(cloudTrailPrincipal) });
+      const logsRole = new iam.Role(this, 'LogsRole', { assumedBy: new iam.ServicePrincipal(cloudTrailPrincipal) });
+      this.cloudWatchLogsRoleArn = logsRole.roleArn;
```

Comment:
Why is `cloudWatchLogsArn` optional?

Reply:
Because it is meaningless if you do not enable output in CloudWatch Logs.